test/xpu_api/utilities/pairs/pair.astuple/tuple_element_pair_as_tuple.pass.cpp

```diff
@@ -0,0 +1,104 @@
+// -*- C++ -*-
+//===----------------------------------------------------------------------===//
+//
+// Copyright (C) Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+// This file incorporates work covered by the following copyright and permission
+// notice:
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+//
+//===----------------------------------------------------------------------===//
+
+#include "support/test_config.h"
+
+#include <oneapi/dpl/type_traits>
+#include <oneapi/dpl/utility>
+
+#include "support/test_macros.h"
+#include "support/utils.h"
+
+#if TEST_DPCPP_BACKEND_PRESENT
+template <typename T1, typename T2>
+bool
+test()
+{
+
+    bool result = false;
+    {
+        typedef T1 Exp1;
+        typedef T2 Exp2;
+        typedef dpl::pair<T1, T2> P;
```

Comment:
```suggestion
        typedef dpl::pair<T1, T2> P;
```

Same for other places in this file.